### ISSUE TITLE
Hand paresthesia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6726]
**Preferred term label:**
Hand paresthesia

**Synonyms**
Hand numbness, Hand tingling

**Definition (free text, please give PubMed ID)**
Tingling (often refered to as a pins and needles feeling) and numbness in the hand.

**Parent term (use hpo.jax.org/app)**
Acroparesthesia HP:0031006

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619161

14/14 PMID:32686688

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
